content/en/community/end-user/discussion-group.md

```diff
@@ -5,46 +5,8 @@ description: Monthly OpenTelemetry usage discussions, all are welcome!
 weight: 20
 ---
 
-**As of April 2024, AMER and EMEA sessions are discontinued in favor of
-discussion panels with a more targeted focus.**
+**As of April 2024, discussion group sessions are discontinued in favor of
+end-user panels with a more targeted focus.**
```

Comment:
```suggestion
**As of April 2024, discussion-group sessions are discontinued in favor of
end-user panels with a more targeted focus.**
```

Reply:
I'm not sure about this one. Would it be a good practice to hyphen "discussion group" here?

Reply:
We hyphenate "discussion-group" for the same reason we hyphenate "end-user" (later in the sentence) -- both are compound adjectives.